api/src/main/java/org/openmrs/scheduler/SchedulerUtil.java

```diff
@@ -141,7 +141,8 @@ public static void sendSchedulerError(Throwable throwable) {
 	public static String getExceptionAsString(Throwable t) {
 		StringWriter stringWriter = new StringWriter();
 		PrintWriter printWriter = new PrintWriter(stringWriter, true);
-		t.printStackTrace(printWriter);
+		log.error(t);
+		//		t.printStackTrace(printWriter);
 		printWriter.flush();
 		stringWriter.flush();
 		return stringWriter.toString();
```

Comment:
This should not call log.error(t). It should return ExceptionUtil.getStrackTrace(t);